oval_graph/client.py

```diff
@@ -162,6 +171,11 @@ def parse_arguments(self, args):
             action="store_true",
             default=False,
             help="It does not start the web browser.")
+        parser.add_argument(
+            '--out',
+            action="store",
+            default=False,
```

Comment:
If you don't specify the default, it will be `None`, which is probably good enough.